modules/assim.batch/R/pda.emulator.R

```diff
@@ -450,19 +487,27 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
   
   for (c in seq_len(settings$assim.batch$chain)) {
     
-    m <- mcmc.out[[c]]$mcmc.samp
+    m <- matrix(NA, nrow =  nrow(mcmc.out[[c]]$mcmc.samp), ncol = length(prior.ind.all.ns))
```

Comment:
For `mcmc` objects from the `coda` package, there are handy `coda::niter` and `coda::nchain` functions that may be somewhat more robust. Just a thought, no need to change it here...